### DISCUSSION TITLE
Question: Is it OK if I create a snap of it?

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1541] [Q&A]
## What is your question?
Am I able to create a snap of FNF?
_Note: Snap is a universal package management system for Linux systems. It is installed by default in Ubuntu and is normally accessed through a graphical store. This will make it easier for Linux users to play. Snap uses a contained environment to run._
It will say that it's unofficial

**Comment by ghost** [comment]
This will be a snap of the latest version of Friday Night Funkin' straight from here.

**Comment by ghost** [comment]
or is it allowed to be the official snap? it will allow more users to try the game
_note: will probably use latest stable for default_

**Comment by ghost** [comment]
it would be nice for linux users but how will you do that

**Comment by ghost** [comment]
1. compile for Linux since i have Ubuntu and use it mainly
2. create a snap details file, build snap
3. publish to snap store

**Comment by Ademking** [comment]
@VMGuy23 I think @ninjamuffin99 should do this.. not anyone else..
because anyone can upload and inject malwares hidden in game.

**Comment by OPNA2608** [comment]
Packaging FNF for Nixpkgs rn. I can add an "unofficial build" disclaimer to the description or make it a glorified AUR script that builds everything on the user's machine instead of our build servers if anyone's concerned about stuff like that.

https://github.com/NixOS/nixpkgs/pull/116800

**Comment by ghost** [comment]
snaps are meant to be kind of malware-proof: its a contained environment

**Comment by Windows1Door2Daniel** [comment]
is more safe put on gamebanana

someone scan your snap

**Comment by ghost** [comment]
Gamebanana goes at around 300kbps download for me
